templates/etc/init.d/rpm/td-agent

```diff
@@ -129,9 +129,6 @@ START_STOP_DAEMON_ARGS="${START_STOP_DAEMON_ARGS}"
 # Exit if the package is not installed
 [ -x "${TD_AGENT_RUBY}" ] || exit 0
 
-# Source function library.
-. <%= File.join(root_path, "etc/init.d/functions") %>
```

Comment:
We should keep track of all the deprecation-specific changes for a future bug/tech debt elimination PR, otherwise casual observers will wonder why library imports differ in platforms where they don't necessarily differ that much.

In other words, any change which is more RHEL5 vs RHEL6+/SuSE; as opposed to SuSE-specific should be noted.

Reply:
This is for SuSE, which doesn't have sysvinit functions. However, it also makes sense as a cleanup (for uniformity). We will test it on other platforms and merge this branch as-is into `master` if it works everywhere.